### PR TITLE
Wpa supplicant bug fix

### PR DESCRIPTION
**Pull request by pgherghetta**

### CODE REVIEW

**Comment by pgherghetta** [comment]
Wrong files.

File: pkg/wifi/iwl.go [hunk 1]
```diff
@@ -150,66 +150,47 @@ func (w *IWLWorker) GetID(stdout, stderr io.Writer) (string, error) {
...
+ 
+         // Each request has a 30 second window to make a connection
+         ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+         defer cancel()
+         c := make(chan error, 1)
+ 
+         // There's no telling how long the supplicant will take, but on the other hand,
+         // it's been almost instantaneous. But, further, it needs to keep running.
+         go func() {
+                 cmd := exec.Command("wpa_supplicant", "-i"+w.Interface, "-c/tmp/wifi.conf")
```
Comment (rjoleary) [inline pkg/wifi/iwl.go:171]: Put this line change into it's own PR.
